app/views/research_sessions/preview.html.erb

```diff
@@ -193,6 +212,7 @@
     my childʼs data will be used as explained above.
   </p>
 
+
```

Comment:
To be super fussy, this line break isn't needed.

Reply:
I'm surprised rubocop hasn't picked it up. I'll get rid.